### ISSUE TITLE
Frame rate capping option in Steam overlay.

### ISSUE THREAD

**Issue by SteamHawkeye** [ValveSoftware/steam-for-linux#6117]
#### Your system information

* Steam client version (build number or date): Feb 18 2019
* Distribution (e.g. Ubuntu): Linux mint 19.1
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Dear Team Steam,
I love how you guys have made games available on Linux Platform.
I am running Linux Mint 19.1 and the latest version of Proton.
The game runs well, at about the same fps i was getting on Windows platform. I also appreciate that Steam has an option to show fps on the top left corner.
I have a request to make, kindly provide an option to limit the fps just like in Riva tuner statistics server, so that we can play games at 30 fps capped just like on windows. That would be an great Option for buttery smooth gameplay.

Thank you in advance. I believe in you guys.m/)

#### Steps for reproducing this issue:

1. 
2. 
3.

**Comment by kisak-valve** [comment]
Hello @SteamHawkeye, as far as I know, there isn't a reliable OS-level method to limit framerate for Steam to tie into. You might be able to use libstrangle to get the behavior you're looking for.

**Comment by SteamHawkeye** [comment]
I have tried Libstrangle for Steam. It works for majority of games but i am struggling with Strange Brigade, SET LAUNCH OPTION command "VSYNC=0 strangle 30 %command%" doesn't work for Strange Brigade.

**Comment by EnriqueWood** [comment]
What is the current state of this feature? Is it in development or something already? It would be really nice to have in the steam client

**Comment by BloodyIron** [comment]
Yeah can we please get this as a built-in feature? This really would be appreciated!
